env/shot.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+#
+
+description: Toolkit configuration to be loaded whenever an association with a
+             shot has been established.
+
+includes:
+- includes/houdini/shot.yml
+- includes/maya/shot.yml
+- includes/nuke/shot.yml
+- includes/desktop/project.yml
```

Comment:
i think it's fine the way it is! Desktop doesn't really have a concept of a shot context.